frontend/src/metabase/parameters/components/ParametersList/types.ts

```diff
@@ -0,0 +1,28 @@
+import type {
+  DashboardFullscreenControls,
+  DashboardHideParametersControls,
+  DashboardThemeControls,
+} from "metabase/dashboard/hoc/controls";
+import type Question from "metabase-lib/v1/Question";
+import type { Dashboard, Parameter, ParameterId } from "metabase-types/api";
+
+export type ParametersListProps = {
+  parameters: Parameter[];
+} & Partial<
+  {
+    className?: string;
+    question?: Question;
+    dashboard?: Dashboard | null;
+    editingParameter?: Parameter | null;
+    isEditing?: boolean;
+    vertical?: boolean;
+    commitImmediately?: boolean;
+    setParameterValueToDefault?: (parameterId: ParameterId) => void;
+    setParameterValue?: (parameterId: ParameterId, value: string) => void;
+    setParameterIndex?: (id: ParameterId, index: number) => void;
+    setEditingParameter?: (id: ParameterId) => void;
+    enableParameterRequiredBehavior?: boolean;
+  } & Pick<DashboardFullscreenControls, "isFullscreen"> &
+    Pick<DashboardThemeControls, "isNightMode"> &
+    Pick<DashboardHideParametersControls, "hideParameters">
```

Comment:
Since `isFullscreen` and `isNightMode` is a plain boolean, I think we can inline them.

However, I suppose the intent here is to be aware of a breaking change when `isFullscreen` disappears from the `DashboardFullscreenControls`, so we know that this type has to be updated? i.e. to communicate that the prop is passed down from those components. If so, we can leave this as-is.

Reply:
Exactly - since we're prop-drilling these all the way down to ParametersList, I think it would make more sense to ensure that the type remains the same all the way down the chain